### DISCUSSION TITLE
03/036-%E7%AC%AC%E4%B8%89%E5%8D%81%E5%85%AD%E7%AB%A0-%E8%BF%BD%E9%80%90%E5%B9%BB%E5%BD%B1.html

### DISCUSSION THREAD

**Discussion by giscus** [liuli-moe/to-the-stars#21] [General]
books/03/036-%E7%AC%AC%E4%B8%89%E5%8D%81%E5%85%AD%E7%AB%A0-%E8%BF%BD%E9%80%90%E5%B9%BB%E5%BD%B1
在经历了几个世纪的动荡之后，一个乌托邦式的 AI— 人类政府治理着地球，预示着后稀缺社会的来临和太空殖民的新时代。一次意外的接触却让科技更先进的敌对外星种族打破了和平，这迫使魔法少女们走出幕后，拯救人类文明。在这一切之中，志筑良子，一个普通的女孩，仰望着星空，好奇着她在宇宙中的归所。
https://tts.liuli.moe/03/036-%E7%AC%AC%E4%B8%89%E5%8D%81%E5%85%AD%E7%AB%A0-%E8%BF%BD%E9%80%90%E5%B9%BB%E5%BD%B1.html

**Comment by moermeng** [comment]
好耶！

**Comment by 5424274** [comment]
说到最后，要做科研总还是得靠着别人善兴大发给你点钱，而决定他们善兴程度的将会是你的评价，因为他们根本搞不清楚你在做什么。
原文第四段最后一句，两个”善兴“是不是笔误？貌似没有这种用法。

**Reply by rxliuli** [reply]
也许原本是想说“善心大发”，吾辈不太确定 @andylizi

**Reply by andylizi** [reply]
应该是 “善心大发” 吧
